<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralConfigurationPage</class>
 <widget class="QWidget" name="GeneralConfigurationPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>5</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Apparence</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="m_use_system_color_cb">
         <property name="text">
          <string>Utiliser les couleurs du système</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_use_gesture_trackpad">
         <property name="text">
          <string>Utiliser les gestes du pavé tactile</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_zoom_out_beyond_folio">
         <property name="text">
          <string>Autoriser le dézoom au delà du folio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_use_windows_mode_rb">
         <property name="text">
          <string>Utiliser des fen&amp;êtres (appliqué au prochain lancement de QElectroTech)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="m_use_tab_mode_rb">
         <property name="text">
          <string>Utiliser des onglets (appliqué au prochain lance&amp;ment de QElectroTech)</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Projets</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Sauvegarde automatique des projets (appliqué au prochain lancement de QElectroTech)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="m_save_label_paste">
         <property name="text">
          <string>Ne pas conserver les labels des éléments lors des copier coller</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="m_border_0">
         <property name="text">
          <string extracomment="Choix de l'increment de depart 1 ou 0">Numéroter les colonnes de cartouche à partir de 0 (1 sinon)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="m_autosave_sb">
         <property name="frame">
          <bool>true</bool>
         </property>
         <property name="specialValueText">
          <string>Désactivé</string>
         </property>
         <property name="suffix">
          <string comment="minute"> min</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="m_use_folio_label">
         <property name="text">
          <string>Utiliser les numéros de folio à la place de leur position dans le projet</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="m_export_terminal">
         <property name="text">
          <string>Exporter les bornes dans la nomenclature</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Collections</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Accès aux collections</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Répertoire de la collection commune</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="m_common_elmt_path_cb">
            <item>
             <property name="text">
              <string>Par defaut</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Parcourir...</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Répertoire de la collection utilisateur</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="m_custom_elmt_path_cb">
            <item>
             <property name="text">
              <string>Par defaut</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Parcourir...</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="m_custom_tbt_path_cb">
            <item>
             <property name="text">
              <string>Par defaut</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Parcourir...</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>(Recharger les collections d'éléments pour appliquer les changements)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Répertoire des cartouches utilisateur</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Gestion des éléments</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="m_highlight_integrated_elements">
            <property name="text">
             <string>Mettre en valeur dans le panel les éléments fraîchement intégrés</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Chaque élément embarque des informations sur ses auteurs, sa licence, ou tout autre renseignement que vous jugerez utile dans un champ libre.
Vous pouvez spécifier ici la valeur par défaut de ce champ pour les éléments que vous créerez :</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="m_default_elements_info">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Langues</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QComboBox" name="m_lang_cb"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Appliqué au prochain lancement de QElectroTech</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>421</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Textes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Vous pouvez définir ici l'apparence par defaut des differents textes de QElectroTech</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Textes d'éléments</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="2" column="2">
           <widget class="QSpinBox" name="m_dyn_text_width_sb">
            <property name="minimum">
             <number>-1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Police :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="m_dyn_text_rotation_sb">
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="suffix">
             <string>°</string>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Longueur :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="m_dyn_text_font_pb"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Rotation :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Textes indépendants</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Rotation :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Police :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="m_indi_text_rotation_sb">
            <property name="suffix">
             <string>°</string>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="m_indi_text_font_pb">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_9">
         <property name="title">
          <string>Autres textes</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Police :</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="m_font_pb">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="GrilleClavier">
      <attribute name="title">
       <string>Grille + Clavier</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Grille : 1 - 30</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="2">
           <widget class="QSpinBox" name="DiagramEditor_xGrid_sb">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="Label_Diagram_yGrid">
            <property name="text">
             <string>DiagramEditor yGrid</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="Label_Grid_Tip">
            <property name="text">
             <string>La Grille doite etre active pour pouvoir voir les modifications.</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="DiagramEditor_yGrid_sb">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="Label_Diagram_xGrid">
            <property name="text">
             <string>DiagramEditor xGrid</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Déplacement au clavier : 1 - 30</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_10">
          <item row="0" column="2">
           <widget class="QSpinBox" name="DiagramEditor_xKeyGrid_sb">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="DiagramEditor_yKeyGrid_sb">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="Label_Diagram_Key_xGrid">
            <property name="text">
             <string>DiagramEditor (touche :   gauche / droite)  xGrid</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="Label_Diagram_Key_yGrid">
            <property name="text">
             <string>DiagramEditor (touche :   haut / bas)  yGrid</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Déplacement au clavier avec la touche ALT : 1 - 9</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="1">
           <widget class="QLabel" name="Label_Diagram_Key_xGrid_ALT">
            <property name="text">
             <string>DiagramEditor (touche :   gauche / droite ) xGrid</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="DiagramEditor_yKeyGridFine_sb">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>9</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="Label_Diagram_Key_yGrid_ALT">
            <property name="text">
             <string>DiagramEditor (touche :  haut / bas)  yGrid</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="DiagramEditor_xKeyGridFine_sb">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>9</number>
            </property>
            <property name="singleStep">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>m_use_system_color_cb</tabstop>
  <tabstop>m_use_gesture_trackpad</tabstop>
  <tabstop>m_zoom_out_beyond_folio</tabstop>
  <tabstop>m_use_windows_mode_rb</tabstop>
  <tabstop>m_use_tab_mode_rb</tabstop>
  <tabstop>m_save_label_paste</tabstop>
  <tabstop>m_use_folio_label</tabstop>
  <tabstop>m_export_terminal</tabstop>
  <tabstop>m_border_0</tabstop>
  <tabstop>m_autosave_sb</tabstop>
  <tabstop>m_common_elmt_path_cb</tabstop>
  <tabstop>m_custom_elmt_path_cb</tabstop>
  <tabstop>m_custom_tbt_path_cb</tabstop>
  <tabstop>m_highlight_integrated_elements</tabstop>
  <tabstop>m_default_elements_info</tabstop>
  <tabstop>m_lang_cb</tabstop>
  <tabstop>m_dyn_text_font_pb</tabstop>
  <tabstop>m_dyn_text_rotation_sb</tabstop>
  <tabstop>m_dyn_text_width_sb</tabstop>
  <tabstop>m_indi_text_font_pb</tabstop>
  <tabstop>m_indi_text_rotation_sb</tabstop>
  <tabstop>m_font_pb</tabstop>
  <tabstop>DiagramEditor_xGrid_sb</tabstop>
  <tabstop>DiagramEditor_yGrid_sb</tabstop>
  <tabstop>DiagramEditor_xKeyGrid_sb</tabstop>
  <tabstop>DiagramEditor_yKeyGrid_sb</tabstop>
  <tabstop>DiagramEditor_xKeyGridFine_sb</tabstop>
  <tabstop>DiagramEditor_yKeyGridFine_sb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
